sklearn/linear_model/tests/test_bayes.py

```diff
@@ -56,3 +56,55 @@ def test_toy_ard_object():
     # Check that the model could approximately learn the identity function
     test = [[1], [3], [4]]
     assert_array_almost_equal(clf.predict(test), [1, 3, 4], 2)
+
+
+def test_return_std_bayesian():
```

Comment:
If the tests are the same (might have overlooked something?), why not do them both in the same test?